ynput/OpenPype
# General: Hero version sites recalculation

## Changelog Description
Sites recalculation in integrate hero version did expect that it is integrated exactly same amount of files as in previous integration. This is not the case in many cases, so the sites recalculation happens in a different way, first are prepared all sites from previous representation files, and all of them are added to each file in new representation.

## Testing notes:
1. Enabled publishing of hero versions
2. Publish some 9 frames in a subset as hero version
3. Publish 10 frames to same subset to create hero version
4. Integration should not crash
5. All integrated files in new representations should have same sites
